main.go

```diff
@@ -53,11 +102,28 @@ func getGroupGifs(c *echo.Context) error {
 }
 
 func createGroup(c *echo.Context) error {
-    res := ResponseTemplate{}
-    group := Group{}
+    g := Group{groupSeq, c.Form("name")}
+
+    rC := RedisConnection()
+    defer rC.Close()
+
+    gJson, err := json.Marshal(g)
+    ErrorHandler(err)
+
+    _, err = rC.Do("SET", "group:"+strconv.Itoa(g.Id), gJson)
+    ErrorHandler(err)
 
-    // save to Redis when created
-    res.Content = group
+    reply, err := redis.Int(rC.Do("INCR", "id:groups"))
```

Comment:
Doing this increment only if I make it past the previous potential error after I do `SET group:{id}`